script/create_test_db.sh

```diff
@@ -1,89 +0,0 @@
-#!/bin/bash
```

Comment:
It's just that script/create_test_db.sh should stay where it is because too many things depend on its location.

And changing the file like you did works, but I'd rather not put everything under a giant `if` block and just keep the scripts separate, since they're for different things. :)